AirQualityWeb/AirQualityWeb/AirQualityWeb/Pages/Index.razor

```diff
@@ -1,8 +1,17 @@
 ﻿@page "/"
+@inject NavigationManager NavManager
 
 <PageTitle>Air Quality</PageTitle>
 
 <h1>Air Quality</h1>
 
+<AirQualityWeb.SharedComponents.AQITable NavigateLocationDetails="NavigateToLocationAsync" />
```

Comment:
The changes in this file are what you should do in the Blazor Hybrid app's Index.razor file. But, of course, in the event handler below you'd instead use Shell's navigation stuff, and have it pop up a MAUI UI page of some sort.